src/test/archive/sample_db/archive_db.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
```

Comment:
We're changing this in place, meaning it's going to be a disaster when we try to have both version work together.

Reply:
Why? I had to regenerate sample data as it is so different from develop/compatible that all archive related tests are failing. If we merge mesa to develop this data should be merged, replacing the one in develop